sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/eventsourcedentity/EventSourcedEntityRouter.scala

```diff
@@ -88,6 +93,8 @@ abstract class EventSourcedEntityRouter[S, E, ES <: EventSourcedEntity[S, E]](pr
       context: CommandContext,
       snapshotEvery: Int,
       eventContextFactory: Long => EventContext): CommandResult = {
+    //do I have the metadata here?
```

Comment:
It seems you have it, no? You accessed it on the command context a few lines later.

Reply:
deleted